docs/supportmodules/fourier.rst

```diff
@@ -22,7 +22,7 @@ frequency-domain values are always treated as complex.
 Using the Fourier Module
 ========================
 
-To get an FFT maker, we may use the `~scintillometry.fourier.get_fft_maker`
+To get an FFT maker, we may use the `~scintillometry.fourier.base.get_fft_maker`
```

Comment:
Why this change? You do import `get_fft_maker` in `fourier.__init__`. Or does sphinx complain?

Reply:
Sphinx for some reason fails to properly link if I don't give the definition path.